service-providers/dotmatrix/25Q4-collaboration.md

```diff
@@ -0,0 +1,46 @@
+# Q4 2025 Marketing Proposal - DeGov
+**DotMatrix LLP - Furqan Ahmed**
+
+## Role: Twitter Spaces & Community Engagement
+
+Based on the [Q4 marketing strategy](https://docs.google.com/document/d/1jHRyWp1LHpEdjg23iGyN8l6ErLQHWLZsBpbAmgmMWAI/edit?tab=t.0#heading=h.gct2345syjqo), I'll support DeGov's community engagement through weekly Twitter Spaces and related activities.
+
+---
+
+## Scope of Work
+
+### Twitter Spaces - Weekly
+- Host weekly Twitter Spaces focused on DAO governance
+- Coordinate X Spaces Promotion through Twitter and Telegram
+- Drive awareness for DeGov.AI in the governance community
+
+### Supporting Activities
+- Promote DeGov updates and features through Spaces discussions
+- Coordinate with team on relevant topics and guests for Spaces
+
+---
+
+## Working Structure
+
+**Weekly:**
+- 1x Twitter Space (~2-3 hours including prep and hosting)
+
+**Communication:**
+- Async updates in Telegram
+- Monthly brief summary
+
+---
+
+## Investment
+
+**Monthly Compensation:** $1,000 USD
+- Payment structure: 10% in RING (monthly average price) + 90% in USDC
+- Paid monthly upon submission of monthly report
+
+**Timeline:** Q4 2025 (October - December)
```

Comment:
Suggest the timeline started from Novermber without end date. As long as both parties agree on the monthly report, the collaboration continues. 

There are few days left in October, and we are working on the new collaboration proposal in this month. So for the new collaboration, let's start from November.